### DISCUSSION TITLE
How can I get all avaiable public image id{/name} for planing compute instances?

### DISCUSSION THREAD

**Discussion by alissa-tung** [huaweicloud/terraform-provider-huaweicloud#1966] [Q&A]
seems that the Huawei Cloud console does not provide the infomation

**Comment by jveraduran** [comment]
Hi @alissa-tung! 

I use terraform and reference a custom image on this way 

```
data "huaweicloud_images_image" "bastion" {
  name        = "bastion"
  most_recent = true
}
```

I let you a public module for reference https://github.com/smu-chile/terraform-huawei-bastion-module/blob/master/bastion.tf
